web/src/components/merchant/listing-details/SendItemModal.tsx

```diff
@@ -63,21 +64,25 @@ const IndicateButton = styled(RoundedButton)`
   width: 370px;
 `;
 
+const indicateItemSent = async (commitId: number) => {
+  await completeCommit(commitId);
+  window.location.reload(false);
```

Comment:
Why do you force a page refresh?

Reply:
@hzxcaryn It's to update the commits. I know this is not the best solution. But will just do this first in case I don't have time for the other stuff.